### PR TITLE
Recreate delete your data

### PR DESCRIPTION
**Pull request by nisarhassan12**

### CODE REVIEW

File: packages/delete-your-data/src/components/hero.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <source src="hero.mp4" type="video/mp4" />
+      </video>
+    </div>
+    <h1>Delete Your Data</h1>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit accusamus
+      sint totam eaque quaerat ullam eos at quas quis, illum vitae est tempore
+      molestias praesentium accusantium eligendi maiores quibusdam et.
+    </p>
+  </StyledHero>
```
Comment (shicholas) [inline packages/delete-your-data/src/components/hero.tsx:43]: I'd like to keep the `Footer` component from the `shared-ui` package, and the `mdxLayout` so we can post blog posts for this website.

Comment (nisarhassan12) [reply packages/delete-your-data/src/components/hero.tsx:43]: Ok. I'll keep them :)

**Comment by nisarhassan12** [comment]
@shicholas This is what the current state of the PR looks like. I will try to incoprate the footer and mdxLayout from share-ui at the end. What do you think about the design so far ? and also what other sections should I create ? I initially thought of creating sections with dummy contents similar to the example website you shared but I think it is better to only work on those that we need ?

Peek 2020-11-28 23-31

From the below paragraph on the old landing page we can have a section which indicates the price with maybe is CTA that says get started in just 5$ or something similar? 

image

or maybe have the pricing CTA in the header like the following 

image

**Comment by shicholas** [comment]
This looks amazing thanks a bunch. I like that CTA you highlighted and would like to price this at $10/year. Please let me know when we have the minimum viable merge. Thanks!

**Pull request merged**